File: petManage_app/src/app/model/pet-servico.ts
import { Cliente } from "./cliente";
import { Pet } from "./pet";

export class PetServico {
    idPetServico: number;
    idPet: number;
    pet: Pet;
    idCliente: number;
    cliente: Cliente;
    observacao: string;
    dtPedido: string;
    dtExecucao: string;
    valor: number;
    flPago: boolean;
    flExecutado: boolean;

    constructor() {
        this.idPetServico = 0;
        this.idPet = 0;
        this.pet = new Pet();
        this.idCliente = 0;
        this.cliente = new Cliente();
        this.observacao = "";
        this.dtExecucao = "";
        this.dtPedido = "";
        this.valor = 0;
        this.flExecutado = false;
        this.flPago = false;
    }
}

File: petManage_api/src/main/java/dot/petstore/model/PetServico.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package dot.petstore.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PetServico {
    private int idPetServico;
    private int idPet;
    private Pet pet;
    private Cliente cliente;
    private int idCliente;
    private String dtPedido;
    private String dtExecucao;
    private String observacao;
    private double valor;
    private boolean flPago;
    private boolean flExecutado;
}
